README.rst

```diff
@@ -25,7 +25,7 @@ implementations for:
 - Profiling
```

Comment:
Out of curiosity, what are the implementations for debugging/profiling?

Reply:
I don't think there are any at the moment (?). Unless passing a `CommandQueue` with profiling enabled counts there..

It would be nice to have some fancier profiling for things though :grin: